### DISCUSSION TITLE
Option to disable the attached zoomed GUI display.

### DISCUSSION THREAD

**Discussion by papahuge** [apohl79/audiogridder#618] [Ideas]
Hi, I really love this program!! It has really saved some of my messy projects! But if I could suggest one thing... It would be to have the option to remove the attached zoomed-in GUI display that warps the mouse cursor back to the pop-ed out GUI. That would be really helpful and save a lot of time for me. Cause I end up hovering over that attached zoomed GUI and then lose my mouse cursor, lol. Also it would really help save real-estate space on the screen if the option to disable was available. Unless that option is already available somewhere?

Thanks for all your hard work!

-Gene

**Comment by CaBuffalo** [comment]
This doesn't happen to me in Logic. What's your DAW?

On Fri, Jun 18, 2021 at 6:01 AM Gene Shnayder ***@***.***>
wrote:

> Hi, I really love this program!! It has really saved some of my messy
> projects! But if I could suggest one thing... It would be to have the
> option to remove the attached zoomed-in GUI display that warps the mouse
> cursor back to the pop-ed out GUI. That would be really helpful and save a
> lot of time for me. Cause I end up hovering over that attached zoomed GUI
> and then lose my mouse cursor, lol. Also it would really help save
> real-estate space on the screen if the option to disable was available.
> Unless that option is already available somewhere?
>
> Thanks for all your hard work!
>
> -Gene
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/618>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6ATD7WEVYYYE3TMCLTTTM7TPANCNFSM465UK5GQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
